### DISCUSSION TITLE
How to fix wrong values of curvature at borders?

### DISCUSSION THREAD

**Discussion by ldv1** [pyvista/pyvista#4922] [Q&A] [answered]
Hi,

I use [curvature](https://docs.pyvista.org/version/stable/api/core/_autosummary/pyvista.PolyDataFilters.curvature.html) to compute the curvature at each point of a surface mesh. It is known that the curvature at borders is wrong (and this is what I observe in my tests), and so [CurvaturesAdjustEdges](https://examples.vtk.org/site/Python/PolyData/CurvaturesAdjustEdges/) was introduced  to fix this issue. Would it be possible to have this option in the method curvature, i.e. 

`PolyDataFilters.curvature(curv_type='mean', adjust_edges=False, progress_bar=False)` ?

Thanks !

In the mean time, is there a way to call CurvaturesAdjustEdges from pyvista ?

**Comment by tkoyama010** [comment, marked as answer]
@ldv1 
In my mind, your proposed addition of `adjust_edges` is worth considering adding as a feature, could you raise [Issue: Feature Request](https://github.com/pyvista/pyvista/issues/new?assignees=&labels=feature-request&projects=&template=feature-request.yml)?

> In the mean time, is there a way to call CurvaturesAdjustEdges from pyvista ?

Please wait as we will consider this when we consider adding the feature.

**Comment by ldv1** [comment]
Done.

**Reply by tkoyama010** [reply]
Thanks!
